src/bluefinClient.ts

```diff
@@ -1657,6 +1696,10 @@ export class BluefinClient {
     pageSize: number;
     parentAddress?: string;
   }) => {
+    payload = {
+      ...payload,
+      parentAddress: payload.parentAddress,
+    };
```

Comment:
Do we need this update? Will this change the structure of the payload being returned?